hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/FirstLevelBlockCache.java

```diff
@@ -17,22 +17,42 @@
  */
 package org.apache.hadoop.hbase.io.hfile;
 
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
 import org.apache.hadoop.hbase.io.HeapSize;
+import org.apache.hbase.thirdparty.com.google.common.util.concurrent.ThreadFactoryBuilder;
 import org.apache.yetus.audience.InterfaceAudience;
 
 /**
  * In-memory BlockCache that may be backed by secondary layer(s).
  */
 @InterfaceAudience.Private
-public interface FirstLevelBlockCache extends ResizableBlockCache, HeapSize {
+public abstract class FirstLevelBlockCache implements ResizableBlockCache, HeapSize {
+
+  /* Statistics thread */
+  protected static String STAT_THREAD_ENABLE_KEY = "hbase.lru.stat.enable";
+  protected static boolean STAT_THREAD_ENABLE_DEFAULT = false;
```

Comment:
As said in #3956 , we'd better set it to true in this PR, and file another PR to change to true on master branch.